### PR TITLE
docs: action bar snippets

### PR DESCRIPTION
**Pull request by kat10140**
**Related Issue:** #619 #628 

Do you think this is enough? Or should we be extra explicit and show RTL directions and try and cover all the bases for each comp?

### CODE REVIEW

**Comment by driskull** [comment]
> should we be extra explicit and show RTL directions and try and cover all the bases for each comp?

I don't think we should cover RTL.

I think we should do the basics and cover all the main properties/attributes of the component. Some of them may be pretty basic and not need a snippet.

File: src/components/calcite-action-bar/usage/basic.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+</calcite-action-bar>
+```
+
+#### With grouping
+
+Renders a group of `calcite-actions` contained in a `calcite-action-group`.
+
+```html
+<calcite-action-bar>
+  <calcite-action-group>
```
Comment (driskull) [inline src/components/calcite-action-bar/usage/basic.md]: I would have two groups here and explain that groups will be divided.

File: src/components/calcite-action-bar/usage/basic.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <calcite-action text="Add">
+      <!-- icon -->
+    </calcite-action>
+    <calcite-action text="Save">
+      <!-- icon -->
+    </calcite-action>
+    <calcite-action text="Layers">
```
Comment (driskull) [inline src/components/calcite-action-bar/usage/basic.md]: Maybe layers in a separate group?

File: src/components/calcite-action-bar/usage/basic.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+</calcite-action-bar>
+```
+
+#### Bottom Actions
+
+Renders `calcite-actions` that stick to the bottom of the action bar.
+
+```html
+<calcite-action-bar>
+  <div slot="bottom actions">
```
Comment (driskull) [inline src/components/calcite-action-bar/usage/basic.md]: Maybe have an action at the top too? And just one at the bottom.

Maybe also explain that they will be above the expand/collapse?

File: src/components/calcite-action-bar/usage/basic.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+#### Basic Action Bar (Top Actions)
+
+Renders `calcite-actions` that stick to the top of the bar.
```
Comment (driskull) [inline src/components/calcite-action-bar/usage/basic.md]: Same as Franco's feedback from previous PR: `calcite-action`'s

Comment (driskull) [reply src/components/calcite-action-bar/usage/basic.md]: Occurs multiple places.

**Comment by kat10140** [comment]
dun!

**Comment by kat10140** [comment]
You right 🤓

**Pull request merged**